.eslintrc.json

```diff
@@ -1,10 +1,20 @@
 {
-    "extends": ["./Tools/eslint-config-cesium/browser.js", "prettier"],
+    "extends": [
+        "./Tools/eslint-config-cesium/browser.js", 
+        "prettier",
+        "plugin:es/restrict-to-es5"
+    ],
     "plugins": [
-        "html"
+        "html",
+        "es"
     ],
+    "parserOptions": {
```

Comment:
You shouldn't need this because it's already declared in `browser.js` (unless it's somehow being overwritten somewhere along the way).

Reply:
ah good catch, fixed!